percona_tests/Makefile

```diff
@@ -0,0 +1,48 @@
+#########################
+### tests
+
+# measures avg scrape time and compares old vs new exporters
+test-performance:
+	go test -v -run '^TestPerformance$$' -args -doRun=true
+
+extraMetrics = false
+multipleLabels = false
+dumpMetrics = false
+
+test-metrics:
+	go test -v -run '^TestMissingMetrics$$' -args -doRun=true
+
+test-labels:
+	go test -v -run '^TestMissingLabels$$' -args -doRun=true
+
+test-resolutions-duplicates:
+	go test -v -run '^TestResolutionsMetricDuplicates$$' -args -doRun=true
+
+test-resolutions:
+	go test -v -run '^TestResolutions$$' -args -doRun=true
+
+dump-metrics:
+	go test -v -run '^TestDumpMetrics$$' -args -doRun=true -extraMetrics=$(extraMetrics) -multipleLabels=$(multipleLabels) -dumpMetrics=$(dumpMetrics)
+
+test-consistency: test-metrics test-resolutions test-resolutions-duplicates
+
+#########################
+### env preparation
+
+# download exporter from provided feature build's client binary url
+prepare-exporter:
+	go test -v -run '^TestPrepareUpdatedExporter$\' -args -doRun=true -url=$(url)
+
+prepare-exporter-from-repo:
+	make -C ../ build && cp ../postgres_exporter assets/postgres_exporter
+
+prepare-base-exporter:
+	tar -xf assets/postgres_exporter_percona.tar.xz -C assets/
```

Comment:
its our old exporter (before any updates) to have a baseline.
In tar - because compressed it uses less space

Reply:
yeah, but it adds additional code to the codebase, maybe it's better to keep it not archived? Do we have comparison in file sizes?

Reply:
its 2.2x smaller in archive (15.2m vs 6.8m). but ok, not a problem at all to store uncompressed

Reply:
BTW, can't we substitute it just by some golden file?

Reply:
you mean with stub metrics?

Reply:
also as a second iteration after finishing more important tasks im going to make those tests runnable on mac, because for now they're working only under linux